flashinfer/attention.py

```diff
@@ -169,5 +173,11 @@ def run(
             self._sm_scale,
             *profiler_args,
         )
+        if v_scale is not None:
+            # TODO(Zihao): fused into kernel
+            if out.itemsize == 1:
+                out = (out.to(float) * v_scale).to(out.dtype)
```

Comment:
Can you also implement fusing the multiply inside the kernel in the persistent kernel template?

Also, it will be good to specify an out_dtype, for quantized input such as fp8, we usually use bf16 as output data type. You can check https://github.com/flashinfer-ai/flashinfer/blob/435f34175c5967e88631a31325a4bce46fefe47e/flashinfer/gemm.py#L1198-L1203 for reference.